### PR TITLE
Upgrade alpine and node version

### PR DESCRIPTION
**Pull request by avagraw**
How Built: Built locally using docker desktop. Command used: **docker build -t <local_image_name> -f .\<dockerfile> .**

How verified: For each docker file, validated that all APIs exposed by the testing utility are working as expected. Validated that the unit tests pass.

### CODE REVIEW

File: Dockerfile.arm32v7 [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM arm32v7/node:8-slim
+FROM arm32v7/node:15.5.1-alpine3.10
```
Comment (marianan) [inline Dockerfile.arm32v7:1]: >node:15.5.1-alpine3.10 [](start = 13, length = 22)

Why switching from slim to alpine? We could use this one for example - [https://hub.docker.com/layers/arm32v7/node/10-slim/images/sha256-792ec870d5fc154342c02c9a911adcab24336652cf9f935199d4ef6d8669f610?context=explore](https://hub.docker.com/layers/arm32v7/node/10-slim/images/sha256-792ec870d5fc154342c02c9a911adcab24336652cf9f935199d4ef6d8669f610?context=explore)

File: Dockerfile.amd64 [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM node:8-alpine
+FROM node:15.5.1-alpine3.10
```
Comment (marianan) [inline Dockerfile.amd64:1]: >15.5.1-alpine3.10 [](start = 10, length = 17)

Initially we were using a fixed node major version, why not continue with this approach? For example this image [https://hub.docker.com/layers/amd64/node/15-alpine/images/sha256-7d8fcef4cc84dafe2f4fbcbcd58ee704d452f1e49bc3393c9452992713808dbc?context=explore](https://hub.docker.com/layers/amd64/node/15-alpine/images/sha256-7d8fcef4cc84dafe2f4fbcbcd58ee704d452f1e49bc3393c9452992713808dbc?context=explore)

**Comment by marianan** [comment]
Do we need to update the windows container node version?

**Comment by marianan** [comment]
Please add in your PR description "How built" and "How validated" sections

**Comment by avagraw** [comment]
> Do we need to update the windows container node version?
It is not needed for debugging purposes. However, I can upgrade to the latest version 12 if you think we should stick to the latest?

**Review by marianan — APPROVED** [review]
:shipit:

**Review by smcd253 — APPROVED** [review]

**Pull request merged**